app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/apps/ManageAppsProtectionViewModel.kt

```diff
@@ -40,6 +41,7 @@ import kotlinx.coroutines.ensureActive
 import kotlinx.coroutines.flow.*
 import kotlinx.coroutines.launch
 
+@SuppressLint("NoLifecycleObserver") // we don't use DI here
```

Comment:
unsure if we prefer suppress or replace `DefaultLifecycleObserver` by `MainProcessLifecycleObserver`... I think better to suppress since we are not observing app lifecycle.

Reply:
Suppress yup